.github/workflows/pr-comment.yml

```diff
@@ -17,14 +17,14 @@ jobs:
       github.event.workflow_run.conclusion == 'success'
 
     steps:
-      - name: Download pull request id artifact
+      - name: Download pull request ID artifact
         uses: dawidd6/action-download-artifact@v6
         with:
-          workflow: ${{ github.event.workflow_run.workflow_id }}
+          workflow: test-benchmark.yml
```

Comment:
This is attempt to fix issue on commenting workflow, but as it's run from main branch it's doable to test it before merge...